pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go

```diff
@@ -98,7 +98,7 @@ func main() {
 	//    type: string`
 
 	re := regexp.MustCompile("`([^`]*)`")
-	reNS := regexp.MustCompile(`(?s)\s*namespace:\s*placeholder\s*`)
+	reNS := regexp.MustCompile(`(?s)\s*namespace:\s*\$(SA_NAMESPACE)\s*`)
```

Comment:
I’ll check but I wonder whether this matches — going by the start of the regex, parentheses are the grouping operator, and need to be escaped to match literal parentheses.

Reply:
yup, I forgot to escape the parentheses as well